prestoadmin/topology.py

```diff
@@ -208,6 +220,11 @@ def set_env_from_conf():
 
     env.user = conf['username']
     env.port = conf['port']
+    env.java8_home = None
```

Comment:
Also, please don't set it if the java8_home keyword isn't found. this is what we were talking about at lunch